src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -56,7 +56,7 @@ public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageMo
                 LegacyManifestOutput legacyManifestOutput,
                 TableOfContentsMap tocMap)
         {
-            var rawMetadata = pageModel.Metadata != null ? new JObject(pageModel.Metadata) : new JObject();
+            var rawMetadata = pageModel.Metadata != null ? JObject.FromObject(pageModel.Metadata) : new JObject();
```

Comment:
>JObject.FromObject( [](start = 59, length = 19)

Q: what is the difference?

Reply:
`FromObject` ensures the object is treated as json serialization. `JsonExtensionData` will be populated correctly to the result